### ISSUE TITLE
Options to flush redis cache from admincp

### ISSUE THREAD

**Issue by Xon** [Xon/XenForo2-RedisCache#2]
Sometimes purging the cache is the recommended solution.

- [ ] Button to flush all (via the flushall command)
- [ ] Selective cache invalidation via a redis glob pattern ( scan + del)

**Comment by oskari** [comment]
Is this still relevant to implement?

**Comment by Xon** [comment]
Yes, you'ld need to ensure it targets all the redis compatible backends as XF supports multiple caching backends for various contexts
